optimum/exporters/onnx/config.py

```diff
@@ -338,6 +338,10 @@ def __init__(
             self._encoder_onnx_config = encoder_onnx_config_constructor(config.encoder)
             self._normalized_config.ENCODER_NORMALIZED_CONFIG_CLASS = self._encoder_onnx_config._normalized_config
 
+            self.seq_len_scale = 1
```

Comment:
What does `seq_len_scale` stand for? There is no occurrence of it in transformers code base.